tests/test_compile.py

```diff
@@ -2,15 +2,18 @@
 from copy import copy
 import numpy as np
 
-from myia.api import standard_pipeline
+from myia.api import standard_pipeline, standard_debug_pipeline
 from myia.prim import ops as P
 from myia.prim.py_implementations import \
     typeof, scalar_add, partial
 
 compile_pipeline = standard_pipeline
 
+debug_fn = standard_debug_pipeline \
+    .select('parse', 'resolve', 'infer', 'specialize', 'export')
 
-def parse_compare(*tests, optimize=True, array=False):
+
+def parse_compare(*tests, optimize=True, array=False, python=True):
```

Comment:
The `python` option sort of defeats the point of `parse_compare`, which is to compare the result of the call in Python to the result from Myia. If we are testing the gradient, we should be using `grad_test`.

Reply:
I've added this option mostly for the grad test on the model.  If we end making grad_test work on dataclasses, then this can be removed.